src/openforms/js/components/admin/form_design/registrations/objectsapi/ObjectsApiOptionsFormFields.js

```diff
@@ -0,0 +1,372 @@
+    [objecttypesLoadingError, objecttypeVersionLoadingError]
+  );
+
+  return (
+    <Wrapper>
+      <CustomFieldTemplate
+        id="root_objecttype"
+        label={intl.formatMessage({
+          defaultMessage: 'Objecttype',
+          description: 'Objects API registration options "Objecttype" label',
+        })}
+        rawDescription={intl.formatMessage({
+          defaultMessage:
+            'URL that points to the ProductAanvraag objecttype in the Objecttypes API. The objecttype should have the following three attributes: "submission_id", "type" (the type of productaanvraag) and "data" (the submitted form data)',
+          description: 'Objects API registration options "Objecttype" description',
+        })}
+        rawErrors={getFieldErrors(name, index, validationErrors, 'objecttype')}
+        errors={buildErrorsComponent('objecttype')}
+        displayLabel
+      >
+        {unknownObjecttype || hasApiError ? (
+          <TextInput
+            id="root_objecttype"
+            name="objecttype"
+            value={objecttype}
+            onChange={onFieldChange}
+            allowBlank={true}
+          />
+        ) : (
+          <Select
+            id="root_objecttype"
+            name="objecttype"
+            choices={
+              objecttypesLoading
+                ? LOADING_OPTION
+                : availableObjecttypes.map(ot => [ot.url, `${ot.name} (${ot.uuid})`])
+            }
+            value={objecttype}
+            onChange={onFieldChange}
+            allowBlank={true}
+          />
+        )}
+      </CustomFieldTemplate>
+      <CustomFieldTemplate
+        id="root_objecttypeVersion"
+        label={intl.formatMessage({
+          defaultMessage: 'Objecttype version',
+          description: 'Objects API registration options "Objecttype version" label',
+        })}
+        rawDescription={intl.formatMessage({
+          defaultMessage: 'Version of the objecttype in the Objecttypes API',
+          description: 'Objects API registration options "Objecttype version" description',
+        })}
+        rawErrors={getFieldErrors(name, index, validationErrors, 'objecttypeVersion')}
+        errors={buildErrorsComponent('objecttypeVersion')}
+        displayLabel
+      >
+        {unknownObjecttype || hasApiError ? (
+          <NumberInput
+            id="root_objecttypeVersion"
+            name="objecttypeVersion"
+            value={objecttypeVersion}
+            onChange={onFieldChange}
+            allowBlank={true}
+          />
+        ) : (
+          <Select
+            id="root_objecttypeVersion"
+            name="objecttypeVersion"
+            choices={
+              objecttypeVersionLoading
+                ? LOADING_OPTION
+                : availableObjecttypeVersions.map(otv => [otv.version, otv.version])
+            }
+            value={objecttypeVersion}
+            onChange={onFieldChange}
+            allowBlank={true}
```

Comment:
this should not be optional, a version must be selected. And let's make sure to order the versions from newest -> oldest so that the newest version is selected by default.

That will probably require an explicit `useEffect` with `onChange` call to select a default/initial version